webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
```

Comment:
```suggestion
  <base-card class="follow-list">
```
we give the top level `class` in our components the same name as the component.
we should probably document this, but we can do that in another `PR`